lib/skywalking/util.lua

```diff
@@ -28,6 +28,21 @@ local split = function(str, delimiter)
     return t
 end
 
+
+local checkIgnoreSuffix = function(operationName, ignoreSuffix)
+    if ignoreSuffix ~= nil then
+        local ignore_suffix_table = split(ignoreSuffix, ",")
```

Comment:
This line should move into the `init` stage. `split`ing every time would make performance impact.

Reply:
OK